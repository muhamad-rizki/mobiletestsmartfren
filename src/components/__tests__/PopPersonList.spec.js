/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import {
  PopPersonList,
} from '../index';

describe('PopPersonList Component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <PopPersonList />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
