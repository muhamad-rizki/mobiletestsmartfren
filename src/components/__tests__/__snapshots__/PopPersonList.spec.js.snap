// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopPersonList Component renders as expected 1`] = `ShallowWrapper {}`;

exports[`PopPersonList Component renders as expected with at least one item 1`] = `ShallowWrapper {}`;
